app/uk/gov/hmrc/vatsubscription/config/featureSwitch/FeatureSwitchModel.scala

```diff
@@ -20,7 +20,8 @@ import play.api.libs.json.{Json, OFormat}
 
 case class FeatureSwitchModel(Api1365Version: Api1365Version,
                               stubDes: Boolean,
-                              Api1363Version: Api1363Version)
+                              Api1363Version: Api1363Version,
+                              enableAnnualAccounting: Boolean)
```

Comment:
This also needs adding to `manage-vat-subscription-frontend`. The frontend is how we toggle the feature switches for vat-subscription without having to redeploy the app every time we want to make a change.